db/seeds.rb

```diff
@@ -121,19 +138,22 @@
                              description: "Ah, we're not surprised you've been lured by this dish. It'll make your tastebuds sing.",
                              price: 9.50,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/SirenStew.jpg'))
 
 sphynx         = Item.create(title: "Sphinx Special",
                              description: "A veritable riddle of flavors, this dish is mythicised as merciless and treacherous",
                              price: 30.00,
                              status: "active",
-                             categories: [special_cat])
+                             categories: [special_cat],
+                             image: File.open('public/SphynxSpecial.jpg'))
 
 cyclops        = Item.create(title: "Cyclops Eyes",
                              description: "Unlike the monster, this dish has way more than one",
                              price: 5.00,
                              status: "retired",
-                             categories: [appetizer_cat])
+                             categories: [appetizer_cat],
+                             image: File.open('public/CyclopsEyes.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.